<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDate</class>
 <widget class="QDialog" name="addDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加日程</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/ico/date.ico</normaloff>:/ico/date.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主题：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_theme"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>全天事件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dte_begin">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2019</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>9999</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dte_end">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2019</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>提醒：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_remind">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>-请选择提醒时间-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>活动开始前</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5分钟前</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10分钟前</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15分钟前</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30分钟前</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1小时前</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>重复：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_times">
       <item>
        <property name="text">
         <string>-请选择重复次数-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>一次性活动</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每个工作日</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每周?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每月（?日）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每年（?月?日）</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>时区：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_zone">
       <item>
        <property name="text">
         <string>-请选择您的时区-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>北京(GMT+8:00)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>香港(GMT+8:00)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>澳门(GMT+8:00)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>台北(GMT+8:00)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新加坡(GMT+8:00)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="le_location">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入地点</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_remark">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入备注</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
